### PR TITLE
Fix: profile update and password change

### PR DESCRIPTION
**Pull request by mhizterkeyz**
# Description

Please include a summary of the change and which issue is fixed. Add a refrence to the issue that was fixed! 

## Type of change
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

# Checklist:
- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] My changes generate no new warnin

### CODE REVIEW

File: resources/views/backend/settings/settings.blade.php [hunk 1]
```diff
@@ -191,7 +191,7 @@ function time_relative_to_time($ts) {
                                                     <div class="row">
                                                         <div class="col-md-12">
                                                             <label>Phone Number</label>
-                                                            <input class="form-control" type="text" name="phone_number" value="{{ isset($user_details['phone_number']) ? $user_details['phone_number'] : "not set" }}">
+                                                            <input class="form-control" type="text" name="phone_number" value="{{ isset($user_details['phone_number']) ? $user_details['phone_number'] : "not set" }}" readonly>
                                                         </div>
                                                     </div>
```
Comment (Jeremiahiro) [inline resources/views/backend/settings/settings.blade.php:196]: please let stop having the not set thing.

its either there's a value or not

Comment (mhizterkeyz) [reply resources/views/backend/settings/settings.blade.php:196]: Okay, if there's no value, I just leave it blank, right?

Comment (Jeremiahiro) [reply resources/views/backend/settings/settings.blade.php:196]: yes

Comment (mhizterkeyz) [reply resources/views/backend/settings/settings.blade.php:196]: Okay. On it

**Pull request closed without merging**